packages/FlexGrid/Row/Row.md

```diff
@@ -27,19 +56,31 @@ The `FlexGrid.Row` component can be used for Column alignment and distribution.
 ```jsx
 <FlexGrid>
   <FlexGrid.Row horizontalAlign="center" distribute="around">
-    <FlexGrid.Col span={3}>Hello</FlexGrid.Col>
-    <FlexGrid.Col span={3}>You seem healthy</FlexGrid.Col>
-    <FlexGrid.Col span={3}>Take care</FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Around 1" />
+    </FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Around 2" />
+    </FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Around 3" />
+    </FlexGrid.Col>
   </FlexGrid.Row>
 </FlexGrid>
 ```
 
 ```jsx
-<FlexGrid gutter={false}>
+<FlexGrid>
   <FlexGrid.Row distribute="between">
-    <FlexGrid.Col span={3}>Hello</FlexGrid.Col>
-    <FlexGrid.Col span={3}>You seem healthy</FlexGrid.Col>
-    <FlexGrid.Col span={3}>Take care</FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Between 1" />
+    </FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Between 2" />
+    </FlexGrid.Col>
+    <FlexGrid.Col xs={4} sm={3}>
+      <Input label="Between 3" />
+    </FlexGrid.Col>
```

Comment:
Perhaps we could combine the two `distribute` examples into one example with 2 rows? Then it will take up less space and be contained.